### PR TITLE
Add privacy policy

### PR DESCRIPTION
**Pull request by Altonss**

### CODE REVIEW

**Comment by Altonss** [comment]
> Thank you for your pull request and welcome to our community! We require contributors to sign our [Contributor License Agreement](https://github.com/grote/Transportr/blob/master/CLA.md), and we don't seem to have the user @Altonss on file. In order for your code to get reviewed and merged, please explicitly state that you accept the agreement. Alternatively, you can add a commit that adds yourself to https://github.com/grote/Transportr/blob/master/.clabot

I accept the CLA!

File: privacy-policy.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Privacy Policy
+
+The Transportr developers do not collect any data themselves. But by using the app several personnal informations might get send to third party providers:
```
Comment (ialokim) [inline privacy-policy.md:3]:
```suggestion
The Transportr developers do not collect any data themselves. But by using the app some personal information might get send to third party providers:
```

File: privacy-policy.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Your location is only retrieved for the following 2 usages:
+
+- retrieve the map tiles around your location from the Mapbox API,
+- use the location as a start or end of a trip query.
+
+## Data usage
+
+### Transport information providers
+
+When searching for a trip, the entered data (trip details (starting and ending point, date and time of travel...), your IP address and the fact you are using Transportr) will be available to the local transport provider you selected. For more information about how the selected transport provider stores your data, check their privacy policies.
```
Comment (ialokim) [inline privacy-policy.md:24]:
```suggestion
When searching for a trip, the entered data (trip details, your IP address and the fact you are using Transportr) will be available to the local transport provider you selected. For more information about how the selected transport provider stores your data, check their privacy policies.
```

File: privacy-policy.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+### Map tiles
+
+The map tiles are retrived from the Mapbox API. The data they receive about you when dowloading the map is your IP address, your location and the fact you are using Transportr. For more information about how the selected transport provider stores your data, check their [privacy policy](https://www.mapbox.com/legal/privacy).
```
Comment (ialokim) [inline privacy-policy.md:28]:
```suggestion
The map tiles are retrieved from the Mapbox API. The data they receive about you when downloading the map is your IP address, the location for which you download the map (which might or not be your current location) and the fact you are using Transportr. For more information about how Mapbox uses your data, check their [privacy policy](https://www.mapbox.com/legal/privacy).
```

**Review by ialokim — CHANGES_REQUESTED** [review]
Thanks a lot @Altonss, this looks like a good start at least. I'd just propose some minor fixes (I'm not a native English speaker though).

**Comment by Altonss** [comment]
> Thanks a lot @Altonss, this looks like a good start at least. I'd just propose some minor fixes (I'm not a native English speaker though).

Thanks a lot for the review! It's indeed just a start, but hopefully it will help :)

**Comment by Altonss** [comment]
I accept the CLA :heavy_check_mark:

**Review by ialokim — APPROVED** [review]

**Comment by Altonss** [comment]
Thanks a lot @ialokim !
I hope it could resolve #811 ;) Maybe worth a try submitting it as is, I don't think there is something to loose if they refuse...

**Pull request merged**